ui/core/src/assets.ethereum.mainnet.json

```diff
@@ -419,6 +419,36 @@
       "priceUsd": 1145,
       "symbol": "ETH",
       "transferLimit": "4381237787299667968"
+    },
+    {
+      "address": "0x6fb3e0a217407efff7ca062d46c26e5d60a14d69",
+      "decimals": 18,
+      "imageUrl": "https://assets.coingecko.com/coins/images/3334/small/iotex-logo.png?1547037941",
+      "name": "IoTeX",
+      "network": "ethereum",
+      "priceUsd": 0.0109,
+      "symbol": "IOTX",
+      "transferLimit": "10900000000000000"
+    },
+    {
+      "address": "0x2b89bf8ba858cd2fcee1fada378d5cd6936968be",
+      "decimals": 18,
+      "imageUrl": "https://assets.coingecko.com/coins/images/13767/small/Secret_S_Black_Coingecko.png?1611667298",
+      "name": "Wrapped Secret",
+      "network": "ethereum",
+      "priceUsd": 1.21,
+      "symbol": "WSCRT",
+      "transferLimit": "1210000000000000000"
+    },
+    {
+      "address": "0xFd09Cf7cFffa9932e33668311C4777Cb9db3c9Be",
```

Comment:
Why don't we use regular mana instead of wrapped mana? https://etherscan.io/token/0x0f5d2fb29fb7d3cfee444a200298f468908cc942

Reply:
Confirmed with Jazear that we will be using mana, not wrapped mana.